IPython/nbconvert/filters/highlight.py

```diff
@@ -113,7 +113,7 @@ def _pygments_highlight(source, output_formatter, language='ipython', metadata=N
 
     # If the cell uses a magic extension language,
     # use the magic language instead.
-    if language == 'ipython' \
+    if language in ('ipython', 'ipython3') \
```

Comment:
I think I'd be happy with `language.startswith('ipython')`.

I'm working on the principle that whatever behaviour is desired should be symmetrical for Python 2 and 3. It's possible that there's some situation where it's not right, but if it's the same for both Python versions, we have the best chance of understanding that issue and fixing it in a way that doesn't introduce more issues.